tests/foreman/api/test_repository.py

```diff
@@ -1944,6 +1946,91 @@ def test_positive_sync_publish_promote_cv(self):
         self.assertGreaterEqual(len(result.stdout), 1)
 
 
+class SRPMRepositoryIgnoreContentTestCase(APITestCase):
```

Comment:
Can we add these tests as part of this class ? https://github.com/SatelliteQE/robottelo/blob/c48b3f94bfb33d500408e3d49c9118d48bea1585/tests/foreman/api/test_repository.py#L1739

Reply:
hi @vijay8451, thanks for your review. I would rather to keep them apart since they have different concerns, and different BZs.